### DISCUSSION TITLE
DAB radio     rAudio 20230212

### DISCUSSION THREAD

**Discussion by yves2** [rern/rAudio#1236] [Bug Reports]
when i click on the test 
i only get:
rtsp-simple-server 0.21.2-1
# rtl_test -t
Found 1 device(s):
  0:  Realtek, RTL2838UHIDIR, SN: 00000001

Using device 0: Generic RTL2832U OEM
Found Rafael Micro R820T/2 tuner
Supported gain values (29): 0.0 0.9 1.4 2.7 3.7 7.7 8.7 12.5 14.4 15.7 16.6 19.7 20.7 22.9 25.4 28.0 29.7 32.8 33.8 36.4 37.2 38.6 40.2 42.1 43.4 43.9 44.5 48.0 49.6 
Sampling at 2048000 S/s.
Testing tuner range. This may take a couple of minutes..

and nothing more
whis ssh connection the command 
rAudio:~  #  dab-scanner-rtlsdr -C 5A
find all the radios available

**Comment by Ryo80it** [comment]
Check if the folder /srv/http/data/dabradio exists, otherwise create it manually (give 'http' user and group permissions). Then redo dab scan (/srv/http/bash/dab-scan.sh)
